src/core/Akka.Tests/IO/TcpListenerSpec.cs

```diff
@@ -51,10 +55,41 @@ public async Task A_TCP_Listener_must_continue_to_accept_connections_after_a_pre
             });
         }
 
+        [Fact]
+        public async Task A_TCP_Listener_must_provide_metrics()
+        {
+            await new TestSetup(this, pullMode: false).RunAsync(async x =>
+            {
+                await x.BindListener();
+
+                var socket = await x.AttemptConnectionToEndpoint();
+                await x.Handler.ExpectMsgAsync<Tcp.Connected>();
+
+                var probe = CreateTestProbe();
+                x.Listener.Tell(new Tcp.SubscribeToTcpListenerStats(probe.Ref));
+                var metrics = await probe.ExpectMsgAsync<Tcp.TcpListenerStatistics>();
+
+                Assert.Equal(1, metrics.AcceptedIncomingConnections);
+                //
```

Comment:
Had to comment out all of this code due to https://github.com/akkadotnet/akka.net/issues/7634